randovania/data/json_data/prime3.txt

```diff
@@ -3339,6 +4137,12 @@ Landing Site C
 Asset id: 15514523555396225844
 > Dock to Core Access; Heals? False
   * Normal Door to Core Access/Dock to Landing Site C
+  > Samus Ship
+      Trivial
+
+> Samus Ship; Heals? True
+  > Dock to Core Access
+      Trivial
 
 ----------------
 Core Access
```

Comment:
Add missile > 5 to landing site -> core path.
Connection from core -> landing site is impossible.